src/cmd_all/Cargo.toml

```diff
@@ -20,7 +20,7 @@ ignored = ["workspace-hack", "workspace-config", "task_stats_alloc"]
 
 [dependencies]
 anyhow = "1"
-clap = { version = "4", features = ["derive"] }
+clap = { version = "4", features = ["cargo", "derive"] }
```

Comment:
Where is the `cargo` features used?

Reply:
It's used by `command!` to read `version`, `author`, `about` etc from `Cargo.toml`.